marketplace/picture-optimizer.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "https://schemas.aramis.group/marketplace/picture-optimizer.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "operations": {
+            "type": "object",
+            "additionalProperties": false,
+            "description": "List of operations (filters) to apply on picture",
+            "properties": {
+                "cropping": {
+                    "type": "boolean"
+                },
+                "cropping-margin": {
+                    "type": "boolean"
+                },
+                "cropping-margin-ratio": {
+                    "type": "object",
+                    "description": "List of cropping margin auto",
+                    "additionalProperties": false,
+                    "properties": {
+                        "top" : {
+                            "type": "number"
```

Comment:
What's the unit for this? Pixels? Percent?

Reply:
percent 0 to 1

Reply:
Add `minimum` and `maximum`